examples/turtle.java

```diff
@@ -0,0 +1,53 @@
+///usr/bin/env jbang "$0" "$@" ; exit $?
+
+// Turtle demo using the aplu5 library
+// JBang script to run a Java program that uses the Turtle graphics library
+// Usage: jbang turtle.java
+// Requires: JDK 8 or higher
+
+//DEPS ch.aplu.turtle:aplu5:0.1.9
```

Comment:
i love the idea of turtle example but if it not something available from maven central or that some other way jbang can fetch it (*) then i think it makes more damage than good 

(*) yes, we should probably support //CLASSPATH <url> but we dont right now)